tools/ios-tasks.js

```diff
@@ -489,19 +489,17 @@ async function generatePodfileDepsAsync(
     versionedPodNames.React,
     globals.React.concat(globals.yoga)
   );
-  let podsRootSub = '${PODS_ROOT}';
   let config = `
     # Generated postinstall: ${versionedPodNames.React}
-    if target.pod_name == '${versionedPodNames.React}'
-      target.native_target.build_configurations.each do |config|
+    if pod_name == '${versionedPodNames.React}'
+    target_installation_result.native_target.build_configurations.each do |config|
         config.build_settings['OTHER_CFLAGS'] = [${configValues}]
         config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)']
         config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_DEV=1'
-        # needed for GoogleMaps 2.x
+        config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_ENABLE_INSPECTOR=0'
+        config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}ENABLE_PACKAGER_CONNECTION=0'
```

Comment:
What `ENABLE_PACKAGER_CONNECTION` does? Are you sure it will be versioned? It doesn't start with `RCT` 😉

Reply:
`ENABLE_PACKAGER_CONNECTION` causes RN to open a web socket connection that causes lots of noisy log messages (https://github.com/expo/expo-cli/commit/24bb708bc51b555ef415a30f2baad16c9e8648e2). I seem to remember that it was for special editor integration that might not be useful in open source.

You're right that it is not versioned -- perhaps we should add it to the versioning script and update the ABI31+32 code in `ios/versioned-react-native`.

Reply:
I modified the versioning script so that it namespaces `ENABLE_PACKAGER_CONNECTION` (https://github.com/expo/expo/pull/4258) and also edited the old ABI 31 + 32 code to use the namespaces macros.